README.md

```diff
@@ -48,14 +48,16 @@ pip install -r requirements/base.txt -r requirements/testing.txt -r requirements
 
 - Local Dev with CDK libs
 ```
-pip install -r requirements/base.txt -r requirements/testing.txt -r requirements/local.txt -r requirements/cdk.txt
+pip install -r requirements/base.txt -r requirements/testing.txt -r requirements/local.txt -r requirements/cdk.txt -r cdk/lambdas/wdiv-s3-trigger/requirements.txt
 ```
 
 - Just Running Tests
 ```
 pip install -r requirements/base.txt -r requirements/testing.txt
 ```
 
+- Update requirements
+Bump version in relevant requirements file, e.g. ```base.in``` then run ```python -m invoke requirements --upgrade```.
```

Comment:
Only single backticks required. Also I've a card in asana to document this properly, but this is better than it just being in my head - so thanks!